src/HealthCheck.Mvc/MvcHealthCheckСonfigurator.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web.Hosting;
+using System.Web.Mvc;
+using System.Web.Routing;
+using HealthCheck.Core;
+using HealthCheck.Core.Configuration;
+using HealthCheck.Mvc.ViewDiscovery;
+
+namespace HealthCheck.Mvc
+{
+    public class MvcHealthCheckСonfigurator : ConfiguratorBase
+    {
+        private static MvcHealthCheckСonfigurator _instance;
```

Comment:
Make this `private static readonly` and create instance on same row. No need to wait until `GetInstance()` is called since there is virtually zero overhead in creating the instance.

Reply:
it is used outside